gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinClusterManager.java

```diff
@@ -228,6 +228,8 @@ private void stopAppLauncherAndServices() {
 
   /**
    * Configure Helix quota-based task scheduling
+   * This Task-quota indicates how many helix-tasks can be running concurrently within a Helix Job.
```

Comment:
More precisely: this config controls the number of tasks that are concurrently assigned to a single Helix instance.